perf/specjbb/run/run_multi.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
This shell is set differently to your other shell script

Reply:
agreed 
```suggestion
#!/usr/bin/env bash
```